HW_34/script.ts

```diff
@@ -0,0 +1,102 @@
+
+// Протипизировать данные ниже:
+
+// TASK 1
+
+interface IFaculties {
+  id: number;
+  faculty: string;
+  subjects: string[];
+  countStudents: number;
+}
+const faculties:IFaculties[] = [
+  {
+    id: 1,
+    faculty: "History department",
+    subjects: ["The World History", "History of Rome"],
+    countStudents: 44,
+  },
+  {
+    id: 2,
+    faculty: "Department of Biology",
+    subjects: ["biology", "chemistry"],
+    countStudents: 50,
+  },
+  {
+    id: 3,
+    faculty: "Faculty of Mathematics",
+    subjects: ["mathematics", "geometry", "trigonometry"],
+    countStudents: 72,
+  },
+  {
+    id: 4,
+    faculty: "Faculty of Design",
+    subjects: ["ui", "ux", "graphic design"],
+    countStudents: 37,
+  },
+];
+
+// TASK 2
+
+interface IMovie {
+  id: number;
+  title: string;
+  year: number;
+  released: string;
+  runtime: string;
+  genre: string[];
+  director: string;
+  writer: string;
+  actors: string[];
+  plot: string;
+  country: string;
+  poster:string| unknown;
```

Comment:
"unknown" is unnecessary in this case. Based on the data provided, we know that poster should be a string (yes, the link can be different but it's always a string).

Please fix it and read the article below:
https://javascript.plainenglish.io/what-is-the-unknown-type-in-typescript-5ef6c5333b81